virttest/utils_misc.py

```diff
@@ -1137,32 +1112,7 @@ def is_mounted(src, mount_point, fstype, perm=None, verbose=False,
     :return: if the src is mounted as expect
     :rtype: Boolean
     """
-    if perm is None:
-        perm = ""
-    if fstype_mtab is None:
-        fstype_mtab = fstype
-
-    # Version 4 nfs displays 'nfs4' in mtab
-    if fstype == "nfs":
-        fstype_mtab = "nfs\d?"
-
-    mount_point = os.path.realpath(mount_point)
-    if fstype not in ['nfs', 'smbfs', 'glusterfs', 'hugetlbfs']:
-        if src:
-            src = os.path.realpath(src)
-        else:
-            # Allow no passed src(None or "")
-            src = ""
-    mount_string = "%s %s .*%s %s" % (src, mount_point, fstype_mtab, perm)
-    logging.debug("Searching '%s' in mtab...", mount_string)
-    if verbose:
-        logging.debug("/etc/mtab contents:\n%s", file("/etc/mtab").read())
-    if re.findall(mount_string.strip(), file("/etc/mtab").read()):
-        logging.debug("%s is mounted.", src)
-        return True
-    else:
-        logging.debug("%s is not mounted.", src)
-        return False
+    return utils_disk.is_mount(src, mount_point, verbose, session)
```

Comment:
same as above, fstype_mtab?

Reply:
@CongLi Thanks. The parameter fstype and fstype_mtab is only meaningful when justify is_mounted by file /etc/mtab, while this will not give correct result if only simply mount by command 'mount -type src des', which is what mount() in both utils_disk and utils_misk do. So I inherited method in utils_disk to check is mounted or not by /proc/mounts, in this case, fstype is not really necessary for the check.  In umount(), fstype is also useless besides is_mount(). Only keep these parameters for safe since it is already imported by some other module.